app/services/foreman_rh_cloud/cloud_request_forwarder.rb

```diff
@@ -99,6 +105,10 @@ def connection_test_request?
       ->(request_path) { request_path =~ /redhat_access\/r\/insights\/?$/ }
     end
 
+    def pes_request?
+      ->(request_path) { request_path =~ /api\/pes\// }
```

Comment:
I see it was already there before, but have you considered the `%r{}` notation?
```suggestion
      ->(request_path) { request_path =~ %r{api/pes/} }
```

Also note you only perform partial matching, so `/something/elseapi/pes/` will also match.

Reply:
It comes with the routes filter, so `something/else/api/pes` wouldn't get here anyway.